e2e_tests/test_repos_find.py

```diff
@@ -101,9 +102,11 @@ def test_repos_find(configtype, default):
         args = ["repos", "find", "local", tmpdir]
         if not default:
             args += ["--format", configtype]
+            if exclude:
```

Comment:
Any reason why you make `--exclude` depend on `default`?

`default` should actually be named `default_format` and just specify whether we *explicitly* pass `--format` or not. This should be independent of `--exclude`. So I think you can de-indent this if condition by one level.

This also means that you can drop the `and not default`s further down.

Reply:
I understood the default flag as "default grm command without any other flag" and that's why I did it the same way as the format flag.
It's more clear now, I'm pushing an update to fix this.